pointcloud_processing:
  ros__parameters:
    tf_x: 0.05
    tf_y: 0.02
    tf_z: -0.05
    tf_roll: 0
    tf_pitch: 0
    tf_yaw: 0
    cam_fov_hor: 69
    cam_width: 320
    cam_height: 180



